metrics/expvar/expvar.go

```diff
@@ -45,6 +49,8 @@ type gauge struct {
 	v *expvar.Float
 }
 
+func (g gauge) String() string { return fmt.Sprintf("%v", g.v) }
```

Comment:
Similarly to Name, String isn't part of any of the metrics definitions. Should it be added?